### PR TITLE
Adding common validity checks , part2

### PR DESCRIPTION
**Pull request by mike-w-wilson**
I made a mess of my git history in the original branch, already reviewed in #389. Nick helped me clean it and created this branch. I've added the changelog updates and module name change. I also addressed  [(#389 (comment)) ](https://github.com/broadinstitute/gnomad_methods/pull/389#discussion_r675681995). I will merge this branch in place of mw/sanity_checks and close the other review.

### CODE REVIEW

**Review by jkgoodrich — APPROVED** [review]
LGTM - I only looked over the change log quick since that is the change since I last looked at it

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@
 * Added function `rekey_new_reference` to re-key a Table or MatrixTable with a new reference genome [(#381)](https://github.com/broadinstitute/gnomad_methods/pull/381)
 * Modified `SEXES` in utils/vcf to be 'XX' and 'XY' instead of 'female' and 'male' [(#381)](https://github.com/broadinstitute/gnomad_methods/pull/381)
 * Fix `annotation_type_is_numeric` and `annotation_type_in_vcf_info` [(#379)](https://github.com/broadinstitute/gnomad_methods/pull/379)
+* Changed module `sanity_checks` to `validity_checks`, modified functions `generic_field_check`, `make_filters_expr_dict` (previously `make_filters_sanity_check_expr`), and `make_group_sum_expr_dict` (previously `sample_sum_check`), and added functions `summarize_variant_filters`, `generic_field_check_loop`, `compare_subset_freqs`, `sum_group_callstats`, `summarize_variants`, `check_raw_and_adj_callstats`, `check_sex_chr_metrics`, `compute_missingness`, `vcf_field_check`, and `validate_release_t` [(#389)](https://github.com/broadinstitute/gnomad_methods/pull/389)
```
Comment (ch-kr) [inline CHANGELOG.md:18]: small note, this is 395 now, right?

**Review by ch-kr — APPROVED** [review]
tiny comment but LGTM! (also only took a quick look at changelog and the small change in sum_group_callstats)

File: CHANGELOG.md (hunk 1 above)
Comment (mike-w-wilson) [reply CHANGELOG.md:18]: Yes, good catch, thank you!

**Pull request merged**